src/main/scala/org/hammerlab/guacamole/commands/SomaticStandardCaller.scala

```diff
@@ -113,6 +117,28 @@ object SomaticStandard {
           args.minTumorAlternateReadDepth
         )
 
+      if (args.dbSnpVcf != "") {
+        val adamContext = new ADAMContext(sc)
+        val dbSnpVariants = adamContext.loadVariantAnnotations(args.dbSnpVcf)
+        potentialGenotypes
```

Comment:
does this command have a side effect that is useful? I'm missing how it's having any effect otherwise